osu.Game/Screens/SelectV2/FooterButtonMods.cs

```diff
@@ -378,6 +378,8 @@ private void load(OsuColour colours)
                     }
                 };
             }
+
+            protected override bool OnMouseDown(MouseDownEvent e) => true;
```

Comment:
why is the other thing using `InputBlockingContainer` but this is directly overriding a virtual

Reply:
I was thinking that since the hover part of the unranked badge is implicitly handled by it implements `IHasTooltip`, so I don't think it needs to be wrapped inside `InputBlockingContainer`.

Reply:
i'd rather the mousedown method is used everywhere it can be personally. `InputBlockingContainer` for a single element feels weird.